packages/core/src/components/hotkeys/hotkey.tsx

```diff
@@ -38,6 +50,20 @@ export interface IHotkeyProps {
     group?: string;
 
     /**
+     * When true, invokes `event.preventDefault()` in response to `keydown` and `keyup` events
+     * before the respective `onKeyDown` and `onKeyUp` callbacks are invoked.
```

Comment:
> When true, invokes `event.preventDefault()` before the respective `onKeyDown` and `onKeyUp` callbacks are invoked. Enabling this can simplify handler implementations.

same for `stopPropagation` below.